layers/imagick/Dockerfile

```
@@ -35,9 +36,11 @@ RUN make install
 
 # Compile the php imagick extension
 WORKDIR ${IMAGICK_BUILD_DIR}
-RUN pecl download imagick
-RUN tar xzf imagick-3.4.4.tgz
-WORKDIR ${IMAGICK_BUILD_DIR}/imagick-3.4.4
+# Temporary workaround for getting right imagick both for PHP 7.* and 8.*
+# @see https://github.com/brefphp/extra-php-extensions/pull/143/files#r541977286
+RUN if [[ "${PHP_VERSION}" = "73" ]] || [[ "${PHP_VERSION}" = "74" ]]; then pecl download imagick-3.4.4 && tar xzf imagick-3.4.4.tgz && mv imagick-3.4.4 imagick ; else git clone https://github.com/Imagick/imagick ; fi
+WORKDIR ${IMAGICK_BUILD_DIR}/imagick
+RUN if [[ "${PHP_VERSION}" != "73" ]] && [[ "${PHP_VERSION}" != "74" ]]; then git checkout 132a11fd26675db9eb9f0e9a3e2887c161875206 ; fi
```

Just some small things to reduce the complexity
```suggestion
RUN if [[ "${PHP_VERSION}" != "80" ]]; then pecl download imagick-3.4.4 && tar xzf imagick-3.4.4.tgz --directory ./imagick ; else git clone https://github.com/Imagick/imagick ; fi
WORKDIR ${IMAGICK_BUILD_DIR}/imagick
RUN if [[ "${PHP_VERSION}" = "80" ]]; then git checkout 132a11fd26675db9eb9f0e9a3e2887c161875206 ; fi
```

---

Although need to remember to support PHP 8.1 in case imagick will not provide a new version before new PHP version will be evailable :)